### DISCUSSION TITLE
best way to convert ext4 to xfs?

### DISCUSSION THREAD

**Discussion by mostafa8026** [minio/minio#12539] [Q&A] [answered]
I have 64999 files in a prefix, based on [this issue](https://github.com/minio/minio/issues/12536), it is the ext4 problem. what is the best way to convert ext4 to xfs?

**Comment by harshavardhana** [comment, marked as answer]
There is no automatic way, reformat and reupload everything

**Comment by mostafa8026** [comment]
I used fstransform and everything is good now (after 2 days 😢)
